Solutions/Salesforce Service Cloud/Analytic Rules/Salesforce-PasswordSpray.yaml

```diff
@@ -0,0 +1,31 @@
+id: 64d16e62-1a17-4a35-9ea7-2b9fe6f07118
+name: Brute force attack against user credentials
+description: |
+  'This query searches for failed attempts to log in from more than 15 various users within a 5 minute timeframe from the same source. This is a potential indication of a password spray attack.'
```

Comment:
Description says it is very 5 minutes but the queryFrequency and queryPeriod is 20mins

Reply:
Change completed